src/api-reference/user-provisioning/v4.spend-user-provisioning.md

```diff
@@ -0,0 +1,702 @@
+                            "canPrepare": true,
+                            "canPrepareForApproval": true,
+                            "canReceiveApprovalEmail": true,
+                            "canReceiveEmail": true,
+                            "canSubmit": true,
+                            "canSubmitTravelRequest": true,
+                            "canUseBi": true,
+                            "canViewReceipt": true,
+                            "delegate": {
+                                "employeeNumber": "expenseDelegate"
+                            },
+                            "temporaryDelegatation": {
+                                "temporaryDelegationFromDate": "2020-02-19T03:15:00.000Z",
+                                "temporaryDelegationToDate": "2020-02-19T03:59:00.000Z"
+                            }
+                        }
+                    ]
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:Role": {
+                    "roles": [
+                        {
+                            "roleName": "EXP_USER",
+                            "roleGroups": ["R&D-Dev-Exp", "R&D-QA-Exp"]
+                        }
+                    ]
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:WorkflowPreference": {
+                    "emailStatusChangeOnCashAdvance": true,
+                    "emailAwaitApprovalOnCashAdvance": true,
+                    "emailStatusChangeOnReport": true,
+                    "emailAwaitApprovalOnReport": true,
+                    "promptForApproverOnReportSubmit": true,
+                    "emailStatusChangeOnTravelRequest": true,
+                    "emailAwaitApprovalOnTravelRequest": true,
+                    "promptForApproverOnTravelRequestSubmit": true,
+                    "emailStatusChangeOnPayment": true,
+                    "emailAwaitApprovalOnPayment": true,
+                    "promptForApproverOnPaymentSubmit": true
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:UserPreference": {
+                    "allowCreditCardTransArrivalEmails": true,
+                    "allowReceiptImageAvailEmails": true,
+                    "promptForCardTransactionsOnReport": true,
+                    "autoAddTripCardTransOnReport": true,
+                    "promptForReportPrintFormat": true,
+                    "defaultReportPrintFormat": "DETAILED",
+                    "showTotalOnReport": true,
+                    "showExpenseOnReport": "ALL",
+                    "showInstructHelpPanel": true,
+                    "showImagingIntro": true,
+                    "expenseAuditRequired": "REQUIRED",
+                    "useQuickItinAsDefault": true
+                },
+                "urn:ietf:params:scim:schemas:extension:enterprise:2.0:Payroll": {
+                    "adp": {
+                        "companyCode": "abcd",
```

Comment:
since other areas are meaningful, might want adp to have a more meaningful example.